### PR TITLE
improve error message when dropping a follower

### PR DESCRIPTION
**Pull request by jsteemann**
### Scope & Purpose

Improve error message in case a truncate operation drops a follower.
The error message now also contains the database name in addition to the shard name, plus the reason for why the follower was dropped.
This is a change that only affects an error message, so there is intentionally no CHANGELOG entry for it.

- [x] :hankey: Bugfix (requires CHANGELOG entry)
- [ ] :pizza: New feature (requires CHANGELOG entry, feature documentation and release notes)
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification
- [ ] :book: CHANGELOG entry made

#### Backports:

- [x] No backports required

### Testing & Verification

- [x] This change is a trivial rework / code cleanup without any test coverage.

### CODE REVIEW

File: arangod/Transaction/Methods.cpp [hunk 1]
```diff
@@ -1880,16 +1880,18 @@ Future<OperationResult> transaction::Methods::truncateLocal(std::string const& c
              responses[i].get().statusCode() == fuerte::StatusOK);
         if (!replicationWorked) {
           auto const& followerInfo = collection->followers();
+          LOG_TOPIC("0e2e0", WARN, Logger::REPLICATION)
+              << "truncateLocal: dropping follower " << (*followers)[i]
+              << " for shard" << collection->vocbase().name() << "/" << collectionName
```
Comment (neunhoef) [inline arangod/Transaction/Methods.cpp:1885]: I suggest to put one more space behind the word "shard" here.

**Review by neunhoef — APPROVED** [review]
LGTM. See one small comment, no need to run jenkins again or me to look again after the change.

File: arangod/Transaction/Methods.cpp (hunk 1 above)
Comment (jsteemann) [inline arangod/Transaction/Methods.cpp:1885]:
```suggestion
              << " for shard " << collection->vocbase().name() << "/" << collectionName
```

Comment (jsteemann) [reply arangod/Transaction/Methods.cpp]: Sure, thanks for the hint. I overlooked this initially.

**Review by hkernbach — APPROVED** [review]

**Pull request merged**